packages/matter-node.js/src/examples/BridgedDevicesNode.ts

```diff
@@ -144,6 +147,39 @@ class BridgedDevice {
             });
         }
 
+        const thermostatDevice = new ThermostatDevice();
```

Comment:
Please remove that again. You can still have this just for you locally. We will think about how having examples for more deviceTypes later. Please not mix too much